nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/ConsumeJMS.java

```diff
@@ -86,6 +86,29 @@
             .defaultValue(CLIENT_ACK.getValue())
             .build();
 
+    static final PropertyDescriptor DURABLE_SUBSCRIBER = new PropertyDescriptor.Builder()
+                                                             .name("Durable subscription")
+                                                             .description("If destination is Topic if present then make it the consumer durable.")
```

Comment:
as per other comment, I'd rather link to JMS spec, as users should understand the behaviour from spec IMO.